### ISSUE TITLE
Spigot Ban Methods - Compilation Issues.

### ISSUE THREAD

**Issue by andrewkm** [DiscordSRV/DiscordSRV#1584]
### Expected behavior

Compile.

### Actual behavior

Compile go boom.

### Steps to reproduce

Compile lol.

### Server software and version

N/A

### Checks

- [X] I am __not__ using an outdated version of DiscordSRV.
- [ ] I asked in [DiscordSRV's Discord server](https://discord.discordsrv.com) to see whether this issue is in fact a bug that needs to be fixed.

### Anything else

https://gist.githubusercontent.com/andrewkm/52ca555467ee33b7227af98261f29cb6/raw/6a0f94d051023ce20c5c2d6b6ea4a16a0177db44/gistfile1.txt

TL;DR Spigot changed some ban methods, although, it might be compounded by paper so...
Needs a bit of digging.

I can get more information in a bit if needed.

EDIT: nvm, here it is: https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/commits/f6d8d44a0292cc3251c8e20af6738be08b88016c#src/main/java/org/bukkit/entity/Player.java

**Comment by Vankka** [comment]
ecc05b6b80cf6c8524652b7f6ae88014da71a0f6
